examples/camera-app/linux/include/camera-device.h

```diff
@@ -66,6 +66,8 @@ static constexpr uint16_t kMaxImageRotation          = 359; // Spec constraint
 
 namespace Camera {
 
+inline constexpr const char * kDefaultVideoDevicePath = "/dev/video0";
```

Comment:
We can drop inline here since kDefaultVideoDevicePath is only used in one compiling unit.

If you need kDefaultVideoDevicePath shared across all translation units (for example, you take the variable’s address and compare it), add inline, but we are not in that case.